### ISSUE TITLE
Heroic Trial of the Champion No Badges

### ISSUE THREAD

**Issue by arestavo** [dalaranwow/dalaran-wow#5720]
[//]: # (INFORMATION ABOUT TAGS)
[//]: # (The black and yellow tag is unique. There can only be one label assigned to them - one black and one yellow)
[//]: # (Priority - ... tags are assigned by team members)
[//]: # (Your issue will be reviewed by testers and the developers will fix it in the future.)
[//]: # (Only once the issue got the label [Fixed - On Live Server] you can find the fix on your Gameserver.)
[//]: # (You can increase the relevance of your issue by adding a thumb-up emote http://i.imgur.com/RFouVVi.png)

[//]: # (***************************************************************)
[//]: # (** !!!!!!!!!!!!!DON'T DELETE THIS TEMPLATE ELSE YOUR ISSUE WILL BE CLOSED!!!!!!!!!!!! **)
[//]: # (***************************************************************)

**Current behavior**: Tell us what happens in detail (Where, When, What, Why, Who)
In Heroic mode Trial of the Champion, no badges are awarded when defeating bosses.
**Expected behavior**: Tell us what should happen instead
In retail, badges were awarded upon defeating a boss (looting a chest). This is further reinforced by looking at AtlasLoot, which shows Heroic mode Trial of the Champion bosses dropping a badge.
**Steps to reproduce the problem**:

1. Enter Heroic Trial of the Champion
2. Defeat any/all bosses
3. No badges received

**Include proofs for this behaviour**

You can research this on several pages like http://wowhead.com or http://wowwiki.wikia.com or http://www.youtube.com/
Issues without proofs will be closed and tag On Hold will be added until the proof is provided.

**Include the ID for the game objects, npcs (creatures,pets,minions), spells, items, quests, instances, zones, achievements, skills**

You can research this on several pages like https://wowgaming.altervista.org/aowow/ or https://wowhead.com/

**Include Screenshots from the issue if necessary**

**Comment by alexuktc** [comment]
Confirming. A loooong standing issue along with the heroic achievements in the dungeon.

**Comment by Lothloryen** [comment]
Thank you for the report. This will be fixed with next maintenance.
